### DISCUSSION TITLE
Release v1.6.0 - Confirmation Dialogs & 'Mute Audio When Out Of Focus' Settings Option

### DISCUSSION THREAD

**Discussion by adrian-miasik** [adrian-miasik/unity-pomodoro#36] [Releases]
<img width="640" alt="unity-pomodoro-release-banner-1.6.0" src="https://user-images.githubusercontent.com/12757380/147378642-0841b4a4-8033-4037-a9cd-4aeacb8334ae.png">

# Release Notes
- Added Confirmation Dialogs! (#22)
  - When users take an action that will interrupt/cancel their current timer, they will now be prompted with a yes/no confirmation dialog to confirm their action. This is intended to prevent accidental restarts.
  - <img width="640" alt="unity-pomodoro-release-banner-1.6.0" src="https://user-images.githubusercontent.com/12757380/147377998-ea19460b-5ea7-48cb-85bf-a3eabcda0975.gif">
- Added `Mute audio when out of focus` settings option. (#21)
  - You can toggle the 'mute audio when out of focus' boolean in the Settings tab.
    - When this option is set to (`True` / Right) and this app is running in the background (not in focus), it won't play the alarm / timer completed audio. 
    - By default this is set to `False` for Windows, Mac, Linux
      - Note: UWP / Microsoft Store users have this set to `True` by default since they get prompted with a Windows Notification when the app is not in focus. You can suppress them from Windows end and set this option to (`False` / Left) in the app to still hear the alarm when the timer is completed. Shoutout to Norman from the reviews for this 👍 
- Theme Slider Revision
- Rename Refactor (#26)
- Bugfixes
  - Fixed Sidebar Row Visibility Bug #20 

# Authors Notes
- Microsoft Store version will be uploaded/updated over the next couple days
- If you have any opinions/requests/ideas on something you'd like to see added, please comment away in the discussion
- I recently applied for GitHub sponsors, so those that wish to support this project you can donate via my [GitHub Sponsors](https://github.com/sponsors/adrian-miasik) page ❤️ 
- If you're enjoying the app, please share it with your friends! Getting the word out means a lot. 
- ☃️ Happy Holidays! 👋

# 🛠 Specifications
Project Size (Zipped): ~23-38 MB
Contributor(s): @adrian-miasik

<hr /><em>This discussion was created from the release <a href='https://github.com/adrian-miasik/unity-pomodoro/releases/tag/'>Release v1.6.0 - Confirmation Dialogs & 'Mute Audio When Out Of Focus' Settings Option</a>.</em>

**Comment by Miles-Larson** [comment]
Just downloaded off the windows store! Works really well! I'm currently having an issue with unwanted toggles between the light-mode & dark-mode. If the time I input has any 1's in it, say 15 minutes, the TriggerThemeSwitch is activated. Unsure if others would encounter this but I'm wondering if we can check if the time editing interface is selected before activating theme switches.


Thanks for this great app!

adrian-miasik/unity-pomodoro/UnityPomodoro/Assets/AdrianMiasik/Scripts/Components/HotkeyDetector.cs

**Reply by adrian-miasik** [reply]
Ah yes, great feedback! This looks like a UX oversight on my part, I've created an issue (#37) for this and will aim to have this resolved in the next release. 

But short summary, I think changing the keyboard binding to something more appropriate is ideal.
We can continue this discussion on the issue page here: #37 

Let me know what you think. 
Many thanks!

**Reply by adrian-miasik** [reply]
I also just realized the [Keyboard Controls document](https://github.com/adrian-miasik/unity-pomodoro/wiki/Keyboard-Controls) didn't have the 'Switch Theme' action in the table. 

I've updated that now.
